lib/appsero/src/insights.php

```diff
@@ -483,7 +483,7 @@ public function optout() {
      *
      * @return integer
      */
-    protected function get_post_count( $post_type ) {
+    public function get_post_count( $post_type ) {
```

Comment:
@tareq1988 I had to make it public. I think it will be better to use this class as abstract and try to send tracker data for multiple plugin in a single post request.

Reply:
Yeah the function can be public, but making the class abstract forces everyone to extend the class.